src/Dialogs/StripeDialog.vala

```diff
@@ -112,6 +144,17 @@ public class AppCenter.Widgets.StripeDialog : Granite.Dialog {
         email_entry.primary_icon_name = "internet-mail-symbolic";
         email_entry.tooltip_text = _("Your email address is only used to send a receipt. You will not be subscribed to any mailing list.");
 
+        var email_label = new Gtk.Label (
+            _("Your email address is only used to send you a receipt.") +
```

Comment:
I think this can be more concise since it's directly associated with the email input

```suggestion
            _("Only used to send you a receipt.") +
```